packages/global-styles/src/components/font-pairing-variations/preview.tsx

```diff
@@ -25,130 +22,132 @@ interface Props {
+	const externalFontFamilies = fontFamilies.filter( ( { slug } ) => slug !== SYSTEM_FONT_SLUG );
+	const [ isLoaded, setIsLoaded ] = useState( ! externalFontFamilies.length );
+	const getFontFamilyName = ( targetFontFamily: string ) => {
+		const fontFamily = fontFamilies.find( ( { fontFamily } ) => fontFamily === targetFontFamily );
+		return fontFamily ? fontFamily.name : targetFontFamily;
+	};
+	const textFontFamilyName = useMemo(
+		() => getFontFamilyName( textFontFamily ),
+		[ textFontFamily, fontFamilies ]
+	);
+	const headingFontFamilyName = useMemo(
+		() => getFontFamilyName( headingFontFamily ),
+		[ headingFontFamily, fontFamilies ]
+	);
+	const handleOnLoad = () => setIsLoaded( true );
+	return (
+		<GlobalStylesVariationContainer
+			width={ width }
+			height={ normalizedHeight }
+			containerResizeListener={ containerResizeListener }
+		>
+			<>
+				<div
+					style={ {
+						// Apply the normalized height only when the width is available
+						height: width ? normalizedHeight : 0,
+						width: '100%',
+						background: 'white',
+						opacity: isLoaded ? 1 : 0,
+						cursor: 'pointer',
+					} }
+				>
+					<div
+						style={ {
+							height: '100%',
+							overflow: 'hidden',
+						} }
+					>
+						{ title ? (
+							<HStack
+								spacing={ 10 * ratio }
+								justify="flex-start"
+								style={ {
+									height: '100%',
+									overflow: 'hidden',
+								} }
+							>
+								<VStack spacing={ 1 } style={ { margin: '16px' } }>
+									<div
+										title={ headingFontFamilyName }
+										aria-label={ headingFontFamilyName }
+										style={ {
+											...DEFAULT_FONT_STYLES,
+											color: '#000000',
+											fontSize: '16px',
+											fontWeight: headingFontWeight,
+											fontFamily: headingFontFamily,
+											fontStyle: headingFontStyle,
+										} }
+									>
+										{ headingFontFamilyName }
+									</div>
+									<div
+										title={ textFontFamilyName }
+										aria-label={ textFontFamilyName }
+										style={ {
+											...DEFAULT_FONT_STYLES,
+											color: '#444444',
+											fontSize: '14px',
+											fontWeight: textFontWeight,
+											fontFamily: textFontFamily,
+											fontStyle: textFontStyle,
+										} }
+									>
+										{ textFontFamilyName }
+									</div>
+								</VStack>
+							</HStack>
+						) : (
+							<HStack
+								align="center"
+								justify="flex-start"
+								style={ {
+									height: '100%',
+									overflow: 'hidden',
+									padding: '16px',
+									boxSizing: 'border-box',
+								} }
+							>
+								<div
+									style={ {
+										fontFamily: textFontFamily,
+										fontStyle: textFontStyle,
+										color: '#000000',
+										fontSize: '16px',
+										fontWeight: textFontWeight,
+										lineHeight: '1em',
+										textAlign: 'center',
+									} }
+								>
+									{ translate( 'Default' ) }
```

Comment:
Note that such simple strings often needs a context for the translators to know how to translate it in various languages.